cdc_rsync_server/cdc_rsync_server.cc

```diff
@@ -151,6 +151,7 @@ CdcRsyncServer::CdcRsyncServer() = default;
 CdcRsyncServer::~CdcRsyncServer() {
   message_pump_.reset();
   socket_.reset();
+  socket_finalizer_.reset();
```

Comment:
Nit: This is not needed.

Reply:
Done. Added a comment in the .h file that the order of declaration is important.